sdk/netapp/Microsoft.Azure.Management.NetApp/src/Generated/Models/Backup.cs

```diff
@@ -37,17 +37,19 @@ public Backup()
         /// <param name="id">Resource Id</param>
         /// <param name="name">Resource name</param>
         /// <param name="type">Resource type</param>
+        /// <param name="backupId">backupId</param>
         /// <param name="creationDate">name</param>
         /// <param name="provisioningState">Azure lifecycle management</param>
         /// <param name="size">Size of backup</param>
         /// <param name="label">Label for backup</param>
         /// <param name="backupType">Type of backup adhoc or scheduled</param>
-        public Backup(string location, string id = default(string), string name = default(string), string type = default(string), System.DateTime? creationDate = default(System.DateTime?), string provisioningState = default(string), long? size = default(long?), string label = default(string), string backupType = default(string))
+        public Backup(string location, string id = default(string), string name = default(string), string type = default(string), string backupId = default(string), System.DateTime? creationDate = default(System.DateTime?), string provisioningState = default(string), long? size = default(long?), string label = default(string), string backupType = default(string))
```

Comment:
This looks like a breaking change. Please let us know if this can be resolved. Thank you

Reply:
If you would like to get this change approved, the API review board needs to approve the Swagger that has already been merged. I am contacting Jeffrey on the API Review board to get this process started.

Reply:
Note that is a fix (bug fix)  backupId property was added to backupProperties that was missing from the spec. If unchanged this would cause issues for users using that particular feature.

I believe already reviewed and accepted, note the service is in preview status.
> From the comment, it seems like service was already supporting this property and it was a miss on swagger side. And moreover its a read only property. signing off on the change from ARM side.